doc/operator-guide/deployment.rst

```diff
@@ -3,4 +3,36 @@ Deployment
 ###########
 
 * Database
+
+    Deployments of the Consolidated Database are currently located at
+
+      -  Summit
+      -  USDF (+ dev)
```

Comment:
It's worth noting here that (1) USDF and USDF-dev use the same underlying database and (2) that database is a replica of the summit database.

Reply:
I will add a clarification for this.